regression-tests/sparktkregtests/testcases/frames/frame_timeseries_test.py

```diff
@@ -98,5 +100,74 @@ def test_frame_timeseries_dickey_fuller_constant_trend_squared(self):
         self.assertAlmostEqual(result.p_value, df_ctt_result[1], delta=0.0001)
         self.assertAlmostEqual(result.test_stat, df_ctt_result[0], delta=0.01)
 
+    def test_frame_timeseries_breusch_pagan(self):
```

Comment:
Could you add either some notes about the auxilliary regression and how it's calculated (that is, the section where you use linear regression), or could you give notes to the references we used to determine this.